ts/features/bonusVacanze/store/actions/bonusVacanze.ts

```diff
@@ -0,0 +1,10 @@
+import { ActionType, createAsyncAction } from "typesafe-actions";
+import { BonusList } from "../../types/bonusList";
+
+export const bonusListLoad = createAsyncAction(
```

Comment:
May this could be renamed in something like `avilableBonusList` we may add in future `activeBonusList` so using the generic `bonusList` could be tricky to understand, what do you think?

Reply:
sounds good!